--- a/.config/ags/modules/cheatsheet/main.js
+++ b/.config/ags/modules/cheatsheet/main.js
@@ -1,10 +1,10 @@
-const { Gtk } = imports.gi;
+const { Gdk, Gtk } = imports.gi;
 import Widget from 'resource:///com/github/Aylur/ags/widget.js';
+import Service from 'resource:///com/github/Aylur/ags/service.js';
+import { Keybinds } from "./keybinds.js";
 import { setupCursorHover } from "../.widgetutils/cursorhover.js";
-import PopupWindow from '../.widgethacks/popupwindow.js';
-import Keybinds from "./keybinds.js";
 
-const CheatsheetHeader = () => Widget.CenterBox({
+const cheatsheetHeader = () => Widget.CenterBox({
     vertical: false,
     startWidget: Widget.Box({}),
     centerWidget: Widget.Box({
@@ -13,18 +13,18 @@ const CheatsheetHeader = () => Widget.CenterBox({
         children: [
             Widget.Box({
                 hpack: 'center',
-                className: 'spacing-h-5 cheatsheet-title',
+                className: 'spacing-h-5',
                 children: [
                     Widget.Label({
                         hpack: 'center',
                         css: 'margin-right: 0.682rem;',
-                        className: 'txt-title',
+                        className: 'txt-title txt',
                         label: 'Cheat sheet',
                     }),
                     Widget.Label({
                         vpack: 'center',
                         className: "cheatsheet-key txt-small",
-                        label: "",
+                        label: "󰘳",
                     }),
                     Widget.Label({
                         vpack: 'center',
@@ -62,22 +62,30 @@ const CheatsheetHeader = () => Widget.CenterBox({
     }),
 });
 
-export default (id) => PopupWindow({
-    name: `cheatsheet${id}`,
-    layer: 'overlay',
+const clickOutsideToClose = Widget.EventBox({
+    onPrimaryClick: () => App.closeWindow('cheatsheet'),
+    onSecondaryClick: () => App.closeWindow('cheatsheet'),
+    onMiddleClick: () => App.closeWindow('cheatsheet'),
+});
+export default () => Widget.Window({
+    name: 'cheatsheet',
+    exclusivity: 'ignore',
     keymode: 'exclusive',
+    popup: true,
+    anchor: ['left', 'right', 'top', 'bottom'],
     visible: false,
-    child: Widget.Box({
-        vertical: true,
-        children: [
-            Widget.Box({
+    child: Widget.Scrollable({
+        vscroll: 'automatic',
+        hscroll: 'never',
+        child: Widget.Box({
                 vertical: true,
                 className: "cheatsheet-bg spacing-v-15",
                 children: [
-                    CheatsheetHeader(),
+                    cheatsheetHeader(),
                     Keybinds(),
                 ]
             }),
-        ],
+    
     })
 });
+
