--- a/.config/hypr/custom/general.conf
+++ b/.config/hypr/custom/general.conf
@@ -1,2 +1,28 @@
 # Put general config stuff here
-# Here's a list of every variable: https://wiki.hyprland.org/Configuring/Variables/
\ No newline at end of file
+# Here's a list of every variable: https://wiki.hyprland.org/Configuring/Variables/
+
+input {
+    touchpad {
+        scroll_factor = 0.6
+        sensitivity = 0.5
+        accel_profile = adaptive
+    }
+}
+
+gestures {
+    workspace_swipe_fingers = 3
+}
+
+monitor=,preferred,auto,1.6
+
+input {
+    touchpad {
+        scroll_factor = 0.6
+        sensitivity = 0.6
+        accel_profile = adaptive
+    }
+}
+
+gestures {
+    workspace_swipe_fingers = 3
+}
